conf/modules.config

```
@@ -232,8 +232,13 @@ process {
         ].join(',').replaceAll('(,)*$', "")
         // setting from https://rdrr.io/bioc/dada2/man/mergePairs.html & https://rdrr.io/bioc/dada2/man/nwalign.html & match = getDadaOpt("MATCH"), mismatch = getDadaOpt("MISMATCH"), gap = getDadaOpt("GAP_PENALTY"), missing from the list below is: 'band = -1'
         ext.args2 = [
-            'minOverlap = 12, maxMismatch = 0, returnRejects = FALSE, propagateCol = character(0), trimOverhang = FALSE, match = 1, mismatch = -64, gap = -64, homo_gap = NULL, endsfree = TRUE, vec = FALSE',
-            params.concatenate_reads ? "justConcatenate = TRUE" : "justConcatenate = FALSE"
+            "minOverlap = ${params.asv_minoverlap ?: 12}, maxMismatch = ${params.asv_maxmismatch ?: 0}, propagateCol = character(0), gap = ${params.asv_gap ?: -64}, homo_gap = NULL, endsfree = TRUE, vec = FALSE",
+            params.asv_concatenate_reads == "consensus" ?
+                "returnRejects = TRUE, match = ${params.match ?: 5}, mismatch = ${params.mismatch ?: -6}" :
```

(1) params names need update & (2) defaults are not supposed to be here.
```suggestion
                "returnRejects = TRUE, match = ${params.asv_match}, mismatch = ${params.asv_mismatch}" :
```